### DISCUSSION TITLE
Pressing "z" or "v" while holding mouse in hitobject timeline should match stable behaviour.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#20184] [Ideas] [answered]
In the stable editor, you could go to the middle of the map, click, and press "z" while the mouse button is held in order to select all the hitobjects from that point to the beginning (or press "v" while the mouse button is held to select the hitobjects from that point to the end). Currently, that behavior is not implemented in lazer.

**Comment by peppy** [comment, marked as answer]
Would be helpful if you can include a video showing the expectation, and what you're seeing on lazer.

**Reply by ghost** [reply]
https://user-images.githubusercontent.com/102317848/189244939-99b63d6b-44bb-496d-b795-b6bf6b84c242.mp4


https://user-images.githubusercontent.com/102317848/189244952-1787592f-6a54-4801-95b6-547df0ef0e43.mp4

The videos start by playing through a section of the map, as seen from 0:00-0:15 on the lazer recording and 0:00-0:14 on the stable recording. After this, the mouse clicks the timeline, and the "z" key is pressed to go to the beginning of the map. After deleting the selection using the delete key, the video replays the same first section of the map. In lazer, you can see that a lot of the hitobjects remain, but in stable, you can see that all the hitobjects have been deleted.

**Reply by peppy** [reply]
Thanks. I was having trouble reproducing the issue with my setup because I can't hit 'z' while dragging (for *reasons*).

This should be resolved alongside https://github.com/ppy/osu/issues/16410, as I believe it's the same issue.
